supervisors/sensitivity_analysis_morris/scalarmapi.R

```diff
@@ -0,0 +1,106 @@
+setGeneric("check_repetitions", function(.Object, params) {
+  .Object
+})
+setGeneric("schedule_point", function(.Object, params) {
+  .Object
+})
+setGeneric("get_result", function(.Object, params) {
+  .Object
+})
+setGeneric("mark_as_complete", function(.Object, result) {
+  .Object
+})
+Scalarm = setClass("Scalarm",
```

Comment:
Jaki ma sens posiadanie stylu kodowania jak go nie egzekwujemy? To się potem naprawdę źle czyta, odkąd wróciłem trafiłem już kilka razy na złe wcięcia czy entery x5, to naprawdę nie kosztuje aż tyle zachodu żeby nie robić czegoś takiego.